src/wesnoth.cpp

```diff
@@ -346,7 +346,8 @@ static int handle_validate_command(const std::string& file, abstract_validator&
 	}
 	std::cout << "Validating " << file << " against schema " << validator.name_ << std::endl;
 	lg::set_strict_severity(0);
-	filesystem::scoped_istream stream = preprocess_file(file, &defines_map);
+	//TODO: maybe also allow 'safe' paths here ?
```

Comment:
This is typically used to read the mainline _main.cfg (or maybe an add-on's _main.cfg), so if not allowing safe paths means that it produces different results from actually loading the data into the game, then it should definitely allow safe paths (probably by default).